daemon/src/db.rs

```diff
@@ -760,15 +760,10 @@ mod tests {
 
             self
         }
-
-        fn with_event_id(mut self, id: BitMexPriceEventId) -> Self {
-            self.order.oracle_event_id = id;
-            self
-        }
     }
 
     impl Order {
-        fn dummy() -> Self {
+        fn dummy_db() -> Self {
```

Comment:
I don't understand the value of this rename?

Reply:
The problem is, that the impl blocks in the tests are accessible over all tests. So, if we have a `dummy` function on `Order` / `Cfd` then that dummy will be accessible from any test.
To make it clearer which dummy is used where I started being more explicit about the dummies. Keep the `dummy_...` function private (because that is specific to a test block and should not be re-used across tests because it could cause weird side effects if we change the dummy...) and be more explicit about the name.

We don't have to do this rename, but I would make it a convention to add where the dummy is used.